Misc/NEWS.d/next/Library/2018-04-24-14-25-07.bpo-33349.Y_0LIr.rst

```diff
@@ -0,0 +1 @@
+lib2to3 now recognizes async generators everywhere.
```

Comment:
I think the news entry needs more details -- it's currently rather vague.

Reply:
How about "lib2to3 now recognizes async generators in non-async contexts"?

Reply:
I have no opposition to the current version though the one you proposed also OK.